ramalama/cli.py

```diff
@@ -205,8 +205,8 @@ def configure_arguments(parser):
     parser.add_argument(
         "--runtime",
         default=config.get("runtime"),
-        choices=["llama.cpp", "vllm"],
-        help="specify the runtime to use; valid options are 'llama.cpp' and 'vllm'",
+        choices=["llama-cpp-python", "llama.cpp", "vllm"],
```

Comment:
As stated above, do not give this choice, just stick with llama.cpp and use llama-cpp-python server for serve